src/software/firmware/includes/screen.h

```diff
@@ -18,8 +18,8 @@
 #include <LiquidCrystal.h>
 
 // Internal
-#include "common.h"
 #include "parameters.h"
+#include "pressure_controller.h"
```

Comment:
I'm not a huge fan of this `include`. 
It will be ok when we implement the new display. See https://github.com/covid-response-projects/covid-respirator/issues/107#issuecomment-610484895 and https://github.com/covid-response-projects/covid-respirator/issues/107#issuecomment-610486775